Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:bullseye-slim
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND="noninteractive" apt-get install -y ffmpeg tzdata fontconfig fonts-dejavu xsltproc jq xmltv-util curl bsdextrautils man \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV ETV_FILLER_DOCKER 1
```

Comment:
This var is used to skip the autoupdate process in docker since there are other methods to get docker updates.